### PR TITLE
Feat/export files

### LINKED ISSUE #154: import records

### PR DESCRIPTION
**Pull request by ramirobg94**
Fixes #154 #155 #163

### CODE REVIEW

File: src/state/form/sagas/importRecords/index.js [hunk 1]
```diff
@@ -83,6 +84,7 @@ function* handleImportRecord(params) {
     let importFiles = [];
```
Comment (SteRiccio) [inline src/state/form/sagas/importRecords/index.js]: this could be a const

File: src/state/surveys/selectors.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 import {createSelector} from 'reselect';
 
 const getSurveys = state => state?.surveys?.data || {};
 const getUi = state => state?.surveys?.ui || {};
 
-const getSurveysAsList = createSelector(
-  getSurveys,
-  surveys => Object.values(surveys) || [],
+const getSurveysAsList = createSelector(getSurveys, surveys =>
+  (Object.values(surveys) || []).sort((surveyA, surveyB) =>
```
Comment (SteRiccio) [inline src/state/surveys/selectors.js:9]: doesn't Object.values always return an array?! it should never return an undefined value...

Comment (ramirobg94) [reply src/state/surveys/selectors.js:9]: Captura de Pantalla 2022-09-09 a las 9 53 57

Comment (SteRiccio) [reply src/state/surveys/selectors.js:9]: I mean the return of Object.values... not the parameter you pass to it...
Object.values always returns an array

Comment (SteRiccio) [reply src/state/surveys/selectors.js:9]: ah now I got it... maybe you wanted to do Object.values(surveys || []) !

File: src/infra/fs/index.js [hunk 3]
```diff
@@ -1,24 +1,61 @@
...
 
 const DEFAULT_ENCODING = 'utf8';
 
-const cleanPathWithBase = (path = '') => {
+const clean = (path, sourcePath) => {
+  return `${sourcePath}/${path
+    .replace(`/private${sourcePath}/`, '')
+    .replace(`private${sourcePath}/`, '')
+    .replace(`${sourcePath}/`, '')
+    .replace(`${sourcePath}`, '')}`;
```
Comment (ramirobg94) [inline src/infra/fs/index.js]:
```suggestion
    .replace(sourcePath, '')}`;
```

**Comment by SteRiccio** [comment]
ok for me

**Review by SteRiccio — APPROVED** [review]

**Pull request merged**